contracts/strategies/PermissionedStrategy.sol

```diff
@@ -35,11 +35,12 @@ contract PermissionedStrategy is IStrategy, AccessControl {
         address _vault
     ) {
         // whitelist default assets
-        setAssetWhitelist(poolShortToken, true);
-        setAssetWhitelist(vaultAsset, true);
+        assetWhitelist[poolShortToken] = true;
+        assetWhitelist[vaultAsset] = true;
 
         // make contract deployer the default admin
         _setupRole(DEFAULT_ADMIN_ROLE, msg.sender);
+        _setupRole(WHITELISTER, msg.sender);
```

Comment:
Should we have a function to change this auth later down the line?

Reply:
(incase we wish to migrate)

Reply:
The default admin is able to change other roles. Let me verify that the OZ package comes with these functions by default but I am fairly sure it does.